lib/ledger_sync/adaptors/quickbooks_online/purchase/searcher.rb

```diff
@@ -0,0 +1,61 @@
+module LedgerSync
+  module Adaptors
+    module QuickBooksOnline
+      module Purchase
+        class Searcher < LedgerSync::Adaptors::Searcher
+          def next_searcher
```

Comment:
We could (should?) dry this up by adding this to a class `LedgerSync::Adaptors::QuickBooksOnline::Searcher < LedgerSync::Adaptors::Searcher` and let that class define `next_searcher` and `previous_searcher` and just inherit here.

Reply:
Ya. I can do that, or we can do in a follow up PR